src/Octopurls.Tests/RedirectsURLsTests.cs

```diff
@@ -0,0 +1,103 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using Xunit;
+
+namespace Octopurls.Tests
+{
+    public class RedirectsURLsTests : OctopurlTest
+    {
+        [Fact]
+        public void TestRedirectsURLs()
+        {
+            foreach (var redirect in redirects.Urls)
+            {
+                var url = redirect.Value;
+                Assert.True(
+                    TestURL(url),
+                    $"Couldn't reach [{url}]"
+                    );
```

Comment:
Just wondering if stopping on the first failure is the best result? I've used something along the lines of the following in the past
```
var invalidUrls = redirects.Urls.Where(url => TestUrl(url) == false).ToArray();
if (invalidUrls.Any())
{
    // echo the list to the console so you can see all of them
}
Assert.False(invalidUrls.Any());
```